JuliaInterface/src/convert.c

```diff
@@ -25,6 +25,14 @@ jl_value_t * julia_gap(Obj obj)
     if (obj == False) {
         return jl_false;
     }
+    if (TNUM_OBJ(obj) == T_COMOBJ &&
```

Comment:
The documentation claims that positional objects are supported, too, but here you only check for `T_COMOBJ`. So also `T_POSOBJ` should be here -- and for HPC-GAP also `T_APOSOBJ` and `T_ACOMOBJ`.

Reply:
Added those with the right `#ifdef` (hopefully). The formatting looks a bit weird.